roles/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+multiqc_ngi_repo: https://github.com/ewels/MultiQC_NGI.git
+multiqc_ngi_dest: "{{ root_path }}/sw/multiqc_ngi"
```

Comment:
If the `multiqc_ngi` dir only exists as a placeholder for your checkout, then perhaps you could rename this to `multiqc_ngi_src` or something similar, to make it more clear that no sw is actually there. Perhaps we should even start using a `/lupus/ngi/src/` instead in the future.